iceoryx_posh/test/CMakeLists.txt

```diff
@@ -60,9 +60,9 @@ set(TEST_LINK_LIBS
 
 if(LINUX)
     set(TEST_LINK_LIBS ${TEST_LINK_LIBS} dl )
-    set(TEST_LINKER_FLAGS -Wl,-z,stack-size=${ICEORYX_SINGLE_PROCESS_STACK_SIZE})
+    set(TEST_LINKER_FLAGS -Wl,-z,stack-size=${ICEORYX_POSH_TEST_STACK_SIZE})
```

Comment:
Are the test running on QNX? I am asking since the default stack size in QNX is even smaller than in linux.

Reply:
@dkroenke tested this yesterday. It requires the stacksize to be increased there as well but he was fine with it. Note that part of it is just due to suboptimal API (in general, not only the discovery). Specifically for discovery related limits, the user will not face these problems if she uses the functional API in a reasonable way (i.e. no large containers on the stack).